### ISSUE TITLE
[Tabs] Add `Presence` to `TabContent` to support unmount animation

### ISSUE THREAD

**Issue by jjenzz** [radix-ui/primitives#860]
Looks like we forgot to add `Presence` to this component.

**Comment by asherccohen** [comment]
Hi, I'm not sure what the above means but I would also suggest to not render the content of an inactive tab, or at least make it configurable.

This is a pretty common case for tabs to enhance performance of heavy sub components.

In my code I have to do:

```
        <Tabs
              defaultValue={selectedTab}
            >
              <TabsList>
                <TabsTrigger value="0">Tab 1</TabsTrigger>
                <TabsTrigger value="1">Tab 2</TabsTrigger>
              </TabsList>

              {selectedTab === "0" ? (
                <TabsContent value="0">
                  <MyHeavySubComponent1 />
                </TabsContent>
              ) : null}

              {selectedTab === "1" ? (
                <TabsContent value="1">
                  <MyHeavySubComponent2 />
                </TabsContent>
              ) : null}

            </Tabs>
```

**Comment by benoitgrelard** [comment]
Hi @asherccohen, if I follow you correctly, what you're talking about is the same as described in this issue? #813 
If so, we have already addressed that in #859 and it's released in the latest release candidates. I believe it's available since  [0.1.1-rc.6](https://www.npmjs.com/package/@radix-ui/react-tabs/v/0.1.1-rc.6).

**Comment by benoitgrelard** [comment]
@asherccohen yes that's exactly what we did: https://github.com/radix-ui/primitives/pull/859/files#diff-270239882d345dfbd3b8e546c353876c6d801c253a85b21254b7395792715f26R200
